src/bridge/cockpitrouter.c

```diff
@@ -63,6 +64,121 @@ enum {
 };
 
 typedef struct {
+  JsonObject *config;
+
+  // Contents owned by config
+  gchar **env;
+  gchar **spawn;
+
+  GHashTable *peers;
+
+} DynamicPeer;
+
+/* TODO: GLIB JSON 1.2 has better hash functions
+ * once we are able to bump our dependencies
+ * we should use them instead
+ */
+static guint
+string_array_hash (gconstpointer key)
+{
+  JsonArray *array;
+  guint hash = 0;
+  guint i;
+  guint length;
+
+  g_return_val_if_fail (key != NULL, 0);
+
+  array = (JsonArray *) key;
```

Comment:
Why does this use a ```JsonArray```, instead of a standard ```gchar**``` (possibly generated by a GPtrArray)?